### DISCUSSION TITLE
Animate virtualised list using Framer Motion

### DISCUSSION THREAD

**Discussion by Meemaw** [TanStack/virtual#482] [General]
Hey. Wondering if anyone has successfully animated `virtual` using framer motion? Was following the example on the website, and just adding a very basic animation which seems to not play well with the `absolute` positioning of the top level list element.

**Comment by piecyk** [comment]
Hi, you can avoid absolute positioning via https://github.com/TanStack/virtual/discussions/476#discussioncomment-4724139

**Reply by Meemaw** [reply]
Hey @piecyk, thanks for answer. I was able to get this working with `useWindowVirtualizer`, but not with `useVirtualizer`.

**Reply by piecyk** [reply]
@Meemaw can you provide an codesandbox example, overall both should work 🤔
